extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -53,6 +53,78 @@
   opacity: 0 !important;
 }
 
+.i-amphtml-story-story-attribution {
+  top: 0 !important;
+  left: 0 !important;
+  position: absolute;
+  border: none !important;
+  pointer-events: auto !important;
+  cursor: pointer !important;
+  max-width: 50% !important;
```

Comment:
This looks weak, what if the system layer icons take more space? What if they don't and there's more space available for the attribution?

Should we change this and make a smarter flex layout, that handles sizing and RTL by default?

Reply:
Done. Since it's a bit of an important change at the system layer I made sure to test different variations / scenarios and took a few screenshots. LMK if you see anything out of place :)

![image](https://user-images.githubusercontent.com/5449100/112910367-d3feb780-90b8-11eb-8dec-8b0dc454bbee.png)
![image](https://user-images.githubusercontent.com/5449100/112911212-8d11c180-90ba-11eb-8529-1b618d954bac.png)

![image](https://user-images.githubusercontent.com/5449100/112910662-76b73600-90b9-11eb-8247-5514adad8aef.png)
![image](https://user-images.githubusercontent.com/5449100/112911244-9d29a100-90ba-11eb-8257-7633edc4cbc4.png)

![image](https://user-images.githubusercontent.com/5449100/112911302-bfbbba00-90ba-11eb-8be9-286c5a129abf.png)
![image](https://user-images.githubusercontent.com/5449100/112911323-c9452200-90ba-11eb-9a24-60d5b7d28ef9.png)

![image](https://user-images.githubusercontent.com/5449100/112911356-deba4c00-90ba-11eb-9794-12076ce470b2.png)